jdaviz/configs/imviz/plugins/coords_info/coords_info.vue

```diff
@@ -5,19 +5,19 @@
     </span>
     <div style="display: inline-block; white-space: nowrap; line-height: 14pt; margin: 0; position: absolute; margin-left: 26px; top: 50%; transform: translateY(-50%); -ms-transform: translateY(-50%);">
       <table>
-        <tr>
+        <tr :style="within_bounding_box ? '' : 'color: gray'">
```

Comment:
@kecnry talked about distinguishing between these two inaccuracies, which is not done here:

1. Out of bounds of its own GWCS bounding box. This should apply regardless to link type.
2. Out of bounds of reference data bounding box. This should only apply when link type is WCS.